boards/shields/max31790/doc/index.rst

```diff
@@ -0,0 +1,29 @@
+.. _max31790_shield:
+
+MAX31790 PWM controller shield
```

Comment:
Any link?

Reply:
The shield is custom made, hence there is no official link available.

Reply:
I don't think we should support custom shields, in the same way we don't support custom boards.
Why would this shield be needed ?